docs/guides/references/trade-offs.mdx

```diff
@@ -27,13 +27,8 @@ to have. In a sense they prevent you from writing bad, slow, or flaky tests.
 
 #### Temporary trade-offs:
 
-We have [open issues](https://github.com/cypress-io/cypress/issues) where you
-can find a full list of things Cypress will eventually address, we wanted to
-highlight some of the more important _temporary_ restrictions that Cypress will
-eventually address. [PRs are welcome ;-)](https://on.cypress.io/contributing)
-
-Many of these issues are currently being worked on or are on our
-[Roadmap](/guides/references/roadmap).
+We wanted highlight some _temporary_ restrictions that Cypress hopes to
```

Comment:
Maybe "We want to highlight ..." ?

Some of the content for these links is quite old and doesn't really give the impression that there is a lot of "temporariness" about the restrictions.

Reply:
```suggestion
We want to highlight some _temporary_ restrictions that Cypress hopes to
```